test/unit/cachers/redis.spec.js

```diff
@@ -439,16 +438,17 @@ describe("Test RedisCacher set & get with namespace & ttl", () => {
 			cacher.logger[level].mockClear()
 		);
 
-		cacher.client.setex = jest.fn(() => Promise.resolve());
+		cacher.client.set = jest.fn(() => Promise.resolve());
 	});
 
-	it("should call client.setex with key & data", () => {
+	it("should call client.set with key & data", () => {
 		cacher.set(key, data1);
-		expect(cacher.client.setex).toHaveBeenCalledTimes(1);
-		expect(cacher.client.setex).toHaveBeenCalledWith(
+		expect(cacher.client.set).toHaveBeenCalledTimes(1);
+		expect(cacher.client.set).toHaveBeenCalledWith(
 			prefix + key,
-			60,
-			cacher.serializer.serialize(data1)
+			cacher.serializer.serialize(data1),
+			"EX",
+			60			
```

Comment:
@fernandodevelon please run the lint:fix command to format the code according to the coding style.
![Screenshot from 2023-06-01 13-13-42](https://github.com/moleculerjs/moleculer/assets/451306/ae132210-1632-4a5e-8535-98e10dbfe2ab)

Reply:
OK, I ran it. Thank you.